test/image/mocks/date_axes_period2.json

```diff
@@ -131,6 +131,7 @@
         },
         "xaxis": {
             "minor": { "ticks": "inside" },
+            "ticklabelshiftx": 15,
```

Comment:
Instead of modifying this mock, I suggest we add a new mock and test this on both x and y axes on different sides and reversed axes as well as inside tick labels.
Something like the `ticklabelposition-1` mock.
Thank you!